### PR TITLE
[iOS] Fix Service JSON parsing

### PR DESCRIPTION
**Pull request by TomoLV**
Added `peripheralIdentifier` argument to the `DomainTypesConverter.serviceFromDictionary` method to provide identifier directly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+
+* iOS - Fix Service JSON parsing
```
Comment (mikolak) [inline CHANGELOG.md]:
```suggestion
* Fix issue where accessing characteristic's properties were null on iOS
```

**Review by mikolak — COMMENTED** [review]
Please make sure the automatic tests are passing

**Comment by TomoLV** [comment]
> Please make sure the automatic tests are passing

I can confirm that automatic tests are passing.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## 1.0.1
+
+* Fix issue where accessing characteristic's properties were null on iOS
```
Comment (mikolak) [inline CHANGELOG.md]:
```suggestion
* Fix an issue causing characteristics' properties to be null on iOS
```

**Review by mikolak — DISMISSED** [review]
Made a mistake in earlier suggestion. Once that's fixed go ahead and merge.

**Review by mikolak — APPROVED** [review]

**Pull request merged**